assets/js/vue/FileManager.vue

```diff
@@ -354,6 +354,11 @@ const initDropzone = () => {
     maxFilesize: null,
     clickable: '#upload-file-button',
     timeout: 0,
+    error: function error(file, message) {
+      if (message.code === 401) {
+        myFileManager.$parent.showPopupError(message.message);
```

Comment:
We should probably do a bit more than just show the error message. It'll be confusing for the user.
So maybe instead it should be a message like: "You are no (longer) logged in!" Click OK (button) to login. Or something.
May have to discuss this with the PO.

Reply:
I think the message should be set in API? Regarding the button, i'm not sure whether we need it because this is a edge case scenario

Reply:
Edge case maybe, but when someone receives this message, they don't know what to do. Also the message is a Symfony error message, but we can change it.

If this happens, the only thing a user should be able/can do, is go back to to the login screen, and I think we should help them with that.

Reply:
It's going to be a super edge case once we adjust the parameters back, and extend the garbage collection startup from 8 hrs to 24 hrs.